iceoryx_posh/source/roudi/roudi_process.cpp

```diff
@@ -126,21 +134,154 @@ ProcessManager::ProcessManager(RouDiMemoryInterface& roudiMemoryInterface,
     m_memoryManagerOfCurrentProcess = m_segmentInfo.m_memoryManager;
 }
 
-void ProcessManager::killAllProcesses() noexcept
+void ProcessManager::killAllProcesses(const units::Duration finalKillTime) noexcept
 {
     std::lock_guard<std::mutex> g(m_mutex);
+    cxx::vector<bool, MAX_PROCESS_NUMBER> processStillRunning(m_processList.size(), true);
```

Comment:
Did you mean processes** ?
i.e. `processesStillRunning`

Reply:
@ithier I would leave the name as it is now since you would read it like `processStillRunning[5]` which sound like is process ```5``` still running

Reply:
I'm with @elfenpiff and would keep it as it is.